group-actors/src/main/java/org/sunbird/service/UserServiceImpl.java

```diff
@@ -87,7 +89,10 @@ public Response searchUserByIds(List<String> userIds) throws BaseException {
   public Response getSystemSettings() throws BaseException {
```

Comment:
context need to be passed

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId